headers/types/dungeon_mode/dungeon.h

```diff
@@ -1096,6 +1097,9 @@ struct dungeon {
     undefined field_0x3b73;
     // 0x3B74: Unknown array, likely one entry per monster species
     uint8_t unknown_array_0x3B74[600];
+    // 0x3DCC: The following entries are 4 bytes long and probably an array. I'm not certain
```

Comment:
Rather than saying what this probably is in a comment, can we just change the struct, and specify in the comment that we aren't totally sure?

```c
undefined4 field_0x3dcc[20];
```
```